src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -170,6 +170,82 @@ def workerCleanup(workerCleanupInfo):
         # TODO: logic here for cleanup flag
         shutil.rmtree(workflowDir)
 
+    @staticmethod
+    def probeUsedDisk(workflowID, workDir=None):
+        '''
+        Used to probe the used disk space (on the disk mounting workDir) minus the space used by
```

Comment:
I think it's ok to use "disk" for method names, but in the comments we should be specific.

So "disk" should be "device" or even "block device" since that is Unix kernel parlance.